django/db/models/base.py

```diff
@@ -749,7 +755,10 @@ def save_base(self, raw=False, force_insert=False,
                 sender=origin, instance=self, raw=raw, using=using,
                 update_fields=update_fields,
             )
-        with transaction.atomic(using=using, savepoint=False):
+        parents = meta.parents.items()
```

Comment:
you don't need `.items()` to get a boolean result, just do `meta.parents` in the condition?